tests/foreman/api/test_location.py

```diff
@@ -502,9 +510,18 @@ def test_positive_update_capsule(self):
 
         @Feature: Location - Update
         """
-        proxy = entities.SmartProxy(id=make_proxy()['id']).search()[0]
+        proxy_id_1 = make_proxy()['id']
+        proxy_id_2 = make_proxy()['id']
+        # Add capsules to cleanup list
+        self.addCleanup(capsule_cleanup, proxy_id_1)
```

Comment:
This works but I don't see any advantage on using `self.addCleanup(entities.Org.delete, {'id': org_id})`.